packages/interactor/src/interactor.ts

```diff
@@ -0,0 +1,86 @@
+import { converge } from './converge';
+import { InteractorSpecification } from './specification';
+import { Locator } from './locator';
+import { defaultOptions } from './options';
+import { NoSuchElementError, AmbigousElementError, NotAbsentError } from './errors';
+import { interaction, Interaction } from './interaction';
+
+export class Interactor<E extends Element> {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  private ancestors: Interactor<any>[] = [];
```

Comment:
Just a nitpick, but this would be a more conventional way to type this:

```suggestion
  private ancestors: Array<Interactor<any>> = [];
```